### PR TITLE
Fixes #13854: Make a feature to open ticket based on rudder events

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/13854

### CODE REVIEW

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
Commit modified

**Comment by Fdall** [comment]
notify/bin/rudder-notifyd is fully commented because I dont think we need it anymore,
thought, I havn't fully tested the functionnalities of the plugin and it may have a sense to fork via python and not use a systemd daemon?

File: notify/Makefile [hunk 1]
```diff
@@ -3,7 +3,8 @@
 # make licensed : will build a license limited version of the plugin
 #
 
-FILES = bin/ etc/ share/
+#FILES = bin/ etc/ share/
```
Comment (amousset) [inline notify/Makefile]: why?

File: notify/bin/rudder-notifyd [hunk 2]
```diff
@@ -1,112 +1,114 @@
...
-        elif sys.argv[1] == "restart":
-            d.restart()
-        else:
-            print(usage)
-            sys.exit(2)
-        sys.exit(0)
-    else:
-        print(usage)
-        sys.exit(2)
+# Do not use anymore, deprecated
```
Comment (amousset) [inline notify/bin/rudder-notifyd]: You should remove it.

File: notify/packaging/postinst [hunk 3]
```diff
@@ -6,14 +6,51 @@ CONFFILE="/opt/rudder/etc/notify.conf"
...
+[GLPI]
+on = false
+userToken = <user token>
+apiToken = <api token>
+url = <API url>
+
+EOF
+fi
+
+SYSTEMDFILE="/etc/systemd/system/notify.service"
```
Comment (amousset) [inline notify/packaging/postinst]: the service name should be `rudder-notify`.

File: notify/share/python/glpi.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    self.__dict__ = fields
+
+  def __init__(self, name, content, status=1, urgency=3, impact=3, priority=3):
+    self.name = name
+    self.content = content
+    self.status = status
+    self.urgency = urgency
+    self.impact = impact
+    self.priority = priority
+#    self.entities_id = entities_id
```
Comment (amousset) [inline notify/share/python/glpi.py]: why are the commented.

**Comment by Fdall** [comment]
PR rebased

File: notify/share/python/glpi.py (hunk 4 above)
Comment (Fdall) [reply notify/share/python/glpi.py]: They are not used here, but may be in future updates

File: notify/Makefile (hunk 1 above)
Comment (Fdall) [reply notify/Makefile]: There are no files to put in the etc/ folder, which will result in a failure at compilation time.
Some files comes directly from a postinst execution and not from a copy while installing the plugin.

**Comment by Fdall** [comment]
OK, merging this PR

**Comment by Fdall** [comment]
OK, merging this PR

**Pull request merged**